ivadomed/postprocessing.py

```diff
@@ -292,7 +292,10 @@ def uncertainty(self, thr, suffix):
             uncertainty_path = self.filename_prefix + suffix
             if os.path.exists(uncertainty_path):
                 data_uncertainty = nib.load(uncertainty_path).get_fdata()
-                self.data_pred = mask_predictions(self.data_pred, data_uncertainty < thr)
+                if suffix == "_unc-iou.nii.gz" or suffix == "_soft.nii.gz":
+                    self.data_pred = mask_predictions(self.data_pred, data_uncertainty > thr)
```

Comment:
I was not thinking of using `_soft.nii.gz`.. but it could make sense! And that's clearly documented so.. all good :)